src/NuGet.Core/NuGet.PackageManagement/Resolution/GatherContext.cs

```diff
@@ -74,6 +80,8 @@ public GatherContext()
         /// </summary>
         public bool IsUpdateAll { get; set; }
 
+        public PackageNamespacesConfiguration PackageNamespacesConfiguration { get; }
```

Comment:
Add docs please. 
Call out that it may be null.

Reply:
Addressed.